src/client/apps/edit/components/content/article_layouts/test/series.test.tsx

```diff
@@ -107,13 +107,31 @@ describe("EditSeries", () => {
     expect(props.onChangeArticleAction.mock.calls[0][1].type).toBe("series")
   })
 
-  it("Renders a background image if url", () => {
-    props.article.hero_section = { url: "http://image.jpg" }
-    const component = getWrapper()
-    expect(component.find(FixedBackground).length).toBe(1)
-    expect(component.find(FixedBackground).props().backgroundUrl).toBe(
-      "http://image.jpg"
-    )
-    expect(component.text()).toMatch("+ Change Background")
+  describe("Image and Video Backgrounds", () => {
+    const videoUrl = "http://video.mp4"
+    const imageUrl = "http://image.jpg"
+
+    const render = url => {
+      props.article.hero_section = { url }
+      return getWrapper()
+    }
+
+    it("Passes a background URL to reaction if it contains an image", () => {
+      const component = render(imageUrl)
+      const bg = component.find(FixedBackground)
+      const url = bg.props().backgroundUrl
+      expect(bg.length).toBe(1)
+      expect(url).toBe("http://image.jpg")
+      expect(component.text()).toMatch("+ Change Background")
+    })
+
+    it("Passes a background URL to reaction if it contains a video", () => {
+      const component = render(videoUrl)
+      const bg = component.find(FixedBackground)
+      const url = bg.props().backgroundUrl
+      expect(bg.length).toBe(1)
+      expect(url).toBe(videoUrl)
```

Comment:
AFAICT, as far as positron is concerned this is correct behavior. In reaction we won't actually render a video URL as a background, but Positron doesn't need to know that.